.docker/ci/Dockerfile

```diff
@@ -38,8 +38,8 @@ RUN apt-get -qq update && \
 # NOTE(mlautman): These pip installs are from the ros2 source install instructions. Seeing as not all of them
 #        are installed in the base image I added them here. I have not been able to verify that they are needed
 #        and if they aren't necessary they should be removed.
-RUN python3 -m pip install --upgrade pip && \
-    python3 -m pip install -U \
+RUN python3 -m pip install -q --upgrade pip && \
```

Comment:
Please update to the newest master. I'm not 100% on the best way to do this but one way would be:

```bash
git fetch upstream
git reset --hard upstream/master
git cherry-pick <Your new commits>
```
In this case it would be 
```bash
git fetch upstream
git reset --hard upstream/master
git cherry-pick e2ac85a184eff0dc5f62bc6032e97f979061c47b 5fce9cface567332607a0acf0d786ee35c171068
```

On my local machine this results in a branch that is identical to `AcutronicRobotics:collision-plugin-loader`

Reply:
I've rebased and pulled. Probably did it wrong. Will have a look again in the future.

Reply:
Totally my bad indeed. Should be fine now @mlautman, thanks!